schemas/yield-aggregator/SCHEMA.md

```diff
@@ -0,0 +1,56 @@
+# Yield Aggregators Schema
+
+This is OpenBlock Labs standard Yield Aggregators schema.
+
+## Schema
+
+### Pools
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| protocolName          | The name of the protocol.                                            | string |
+| chainId               | The standard chain id.                                               | number |
+| creationTimestamp     | The timestamp this pool was created.                                 | number |
+| creationBlockNumber   | The block number that this pool was created.                         | number |
+| underlyingTokenAddress| The contract address of the underlying token or deposited token.     | string |
+| underlyingTokenSymbol | The symbol of the underlying token token.                            | string |
+| receiptTokenAddress   | The contract address of the output or receipt token of this pool, if available.| string |
+| receiptTokenSymbol    | The symbol of the receipt token.                                     | string |
+| poolAddress           | The smart contract address of the pool.                              | string |
+| poolSymbol            | The symbol of the pool.                                              | string |
```

Comment:
we need to adopt similar schema like the ones we have for DEXes 

additional columns :

underlyingTokenIndex
underlyingTokenDecimals
receiptTokenDecimals